puma/roc.py

```diff
@@ -574,4 +574,23 @@ def plot_roc(self, **kwargs) -> mpl.lines.Line2D:
                 zorder=2,
                 **kwargs,
             )
+            if elem.n_test is not None:
+                # if uncertainties are available for roc plotting their uncertainty as
+                # a band around the roc itself
+                rej_band_down = (
+                    elem.bkg_rej[elem.non_zero_mask]
+                    - elem.binomial_error()[elem.non_zero_mask]
+                )
+                rej_band_up = (
+                    elem.bkg_rej[elem.non_zero_mask]
+                    + elem.binomial_error()[elem.non_zero_mask]
+                )
+                self.axis_top.fill_between(
+                    elem.sig_eff[elem.non_zero_mask],
+                    rej_band_down,
+                    rej_band_up,
+                    color=elem.colour,
+                    alpha=0.3,
+                    zorder=2,
+                )
```

Comment:
Should we add a unit test for that?

Reply:
I added one for the 1 ratio and 2 ratio scenario